core/docs/load-testing.md

```diff
@@ -0,0 +1,20 @@
+# Load Testing
+
+Firstly, [install Artillery](https://www.artillery.io/docs/get-started/get-artillery).
+
+```sh
+npm install -g artillery@latest
+```
+
+Set the environment variable `ARTILLERY_CLOUD_API_KEY` to your [Artillery Cloud API](https://app.artillery.io/login) key.
```

Comment:
I think it'd be helpful to note what you noted in your PR description, that this is only needed in order to see results in cloud

```suggestion
Optional: set the environment variable `ARTILLERY_CLOUD_API_KEY` to your [Artillery Cloud API](https://app.artillery.io/login) key. This is only needed in order for the results to report to Artillery Cloud.
```

Reply:
Great suggestion! And yes it's expected to fail right now (should have put that in the description). Kalil and I expect that the issue is due to lack of CPU resources as our application is configured to run on 0.5 vCPUs. So we're going to make a ticket to increase the size and see what happens!